shared/common-adapters/copy-text.tsx

```diff
@@ -142,35 +138,34 @@ const styles = Styles.styleSheetCreate(
           overflow: 'hidden',
         },
         isMobile: {
-          minHeight: 40,
```

Comment:
Intentionally removed? if so, delete instead of commenting out, and make sure it doesn't have any side effects.